src/UI/Factory.php

```diff
@@ -336,4 +336,43 @@ public function panel();
 	 **/
 	public function modal();
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *      Links are used navigate to other resources or views of the system. Clicking
+	 *      on a link does not change the systems status.
+	 *   composition: >
+	 *      Link is a clickable, graphically minimal obtrusive control element. It can
+	 *      bear text or other content.
```

Comment:
Maybe add: Links always contain a valid href tag (which should not just contain a hash tag).